gnomad_qc/v4/variant_qc/evaluation.py

```diff
@@ -154,8 +158,80 @@ def create_aggregated_bin_ht(model_id: str) -> hl.Table:
     return agg_ht
 
 
-def main(args):  # noqa: D103
-    hl.init(log="/variant_qc_evaluation.log")
+def get_evaluation_resources(
+    test: bool,
+    overwrite: bool,
+) -> PipelineResourceCollection:
+    """
+    Get PipelineResourceCollection for all resources needed in the variant QC evaluation pipeline.
+
+    :param test: Whether to gather all resources for testing.
+    :param overwrite: Whether to overwrite resources if they exist.
+    :return: PipelineResourceCollection containing resources for all steps of the
+        variant QC evaluation pipeline.
+    """
+    # Initialize variant QC evaluation pipeline resource collection.
+    evaluation_pipeline = PipelineResourceCollection(
+        pipeline_name="variant_qc_evaluation",
+        overwrite=overwrite,
+        pipeline_resources={
+            "annotations/generate_variant_qc_annotations.py --compute-info --split-info": {
+                "info_ht": get_info(split=True)
```

Comment:
would there ever be a case here where test=True for get_info() ?

Reply:
Not in this case, we are only using the full info HT even when testing because it's already created and only annotated onto a MT to exclude AS_lowqual variants